src/services/helpers.ts

```diff
@@ -194,12 +194,19 @@ export const getSortedAndFilteredVocabularyItems = (
   vocabularyItems: VocabularyItem[] | null,
   searchString: string
 ): VocabularyItem[] => {
+  const collator = new Intl.Collator('de-De', { sensitivity: 'base', usage: 'sort' })
+
   const normalizedSearchString = normalizeSearchString(searchString)
 
+  const getNouns = (item: { word: string }) => {
+    const words = item.word.split(' ')
+    return words.find((word: string) => word[0] === word[0].toUpperCase()) as string
+  }
+
   const filteredVocabularyItems = vocabularyItems?.filter(
     item => item.word.toLowerCase().includes(normalizedSearchString) || matchAlternative(item, normalizedSearchString)
   )
-  return filteredVocabularyItems?.sort((a, b) => a.word.localeCompare(b.word)) ?? []
+  return filteredVocabularyItems?.sort((a, b) => collator.compare(getNouns(a), getNouns(b))) ?? []
```

Comment:
If you changed the function declaration above, you can just call the function here with `getNouns(a.word)`